std/special/panic.zig

```diff
@@ -10,7 +10,7 @@ pub fn panic(msg: []const u8, error_return_trace: ?*builtin.StackTrace) noreturn
     @setCold(true);
     switch (builtin.os) {
         // TODO: fix panic in zen.
-        builtin.Os.freestanding, builtin.Os.zen => {
+        builtin.Os.freestanding, builtin.Os.zen, builtin.Os.uefi => {
```

Comment:
is `while (true) {}` the best we can do for panicking under UEFI? is there no logging mechanism or anything?

Reply:
I'm not yet sure if output is dependent upon which uefi toolkit is used.  I am investigating gnu-efi a bit closer, but as I'm a BSD/MIT/Apache person myself, admittedly I know very little about it.

Reply:
In https://nebulae.online/kzig.zig.txt you're calling this `Print` function. Is that function always guaranteed to exist in the UEFI environment? That could be used to print the panic message, at least.

Reply:
Looks like gnu-efi has a ton of the same Intel code as edk-II, so it may be as simple as Print vs _Print. I think it's safe to assume there's a better panic implementation.